### PR TITLE
BC-2521 trigger valididation on password confirmation input

### PR DESCRIPTION
**Pull request by pkleybolte**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-2521

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by psachmann — CHANGES_REQUESTED** [review]
Walked through the steps mentioned in the Jira-Ticket and still get no hint/notification when the provided passwords are different.

**Comment by pkleybolte** [comment]
> Walked through the steps mentioned in the Jira-Ticket and still get no hint/notification when the provided passwords are different.

Works for me in Chrome, Firefox and mobile Chrome. What browser did you use?

File: static/scripts/dataprivacy/user-registration.js [hunk 1]
```diff
@@ -36,6 +48,9 @@ window.addEventListener('DOMContentLoaded', () => {
 		});
 	}
 
+	if (passwordInput) passwordInput.onchange = validatePassword;
+	if (passwordConfirmInput) passwordConfirmInput.onkeyup = validatePassword;
```
Comment (HeftyTea) [inline static/scripts/dataprivacy/user-registration.js:52]: What was the reason to go with onchange, and onkeyup here? Is oninput an option in both cases?

**Review by HeftyTea — APPROVED** [review]
Functionally review OK. Created teacher, created invitation link, tested password (mis)-match OK. Blocked on client side OK. 
Technical review OK. (see question regarding event.

**Review by psachmann — APPROVED** [review]

**Pull request merged**